### DISCUSSION TITLE
How can I disable posting activity comments for all users in collections, users etc

### DISCUSSION THREAD

**Discussion by awnish-npg** [directus/directus#7216] [Archive]
I need to disable the activity comments from my application as it is causing problem in security audit.......

![image](https://user-images.githubusercontent.com/60690093/128358721-a230cda7-95b3-40e5-b3bf-8d881e798571.png)

**Comment by rijkvanzanten** [comment]
> as it is causing problem in security audit.......

What problem?

**Reply by awnish-npg** [reply]
Asking not to allow the HTML tags, javascript tags, so I would prefer rather hide this feature from all other roles. Is there any way to not allow to load the Activity Comments section itself?
Please suggest.

**Reply by rijkvanzanten** [reply]
> Asking not to allow the HTML tags, javascript tags

I don't understand what you mean..

> Is there any way to not allow to load the Activity Comments section itself?

No, the availability of this section can't be configured at this time

**Reply by awnish-npg** [reply]
Thanks. Noted
